examples/fast-likes-smart-feeds/.env.example

```
@@ -0,0 +1,8 @@
+# Supabase Configuration
+SUPABASE_URL=https://your-project.supabase.co
+SUPABASE_ANON_KEY=your-anon-key
+
+# Firebase Configuration
+FIREBASE_SERVICE_ACCOUNT_PATH=/path/to/firebase-service-account.json
+# OR use JSON directly (for deployment)
+FIREBASE_SERVICE_ACCOUNT_JSON={"type":"service_account","project_id":"your-project",...}
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)